src/Quotes.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect, useRef } from 'react';
+// import { FaTwitter } from 'react-icons/fa';
+
+const Quote = () => {
+  const [quotes, setQuotes] = useState([]);
+  const colors = ['#9e4200', '#421c00', '#628500', '#009025', '#391306', '#aa1945', '#3B0918', '#b8390e'];
+  const textRef = useRef();
+  const getQuotes = () => {
+    fetch('https://type.fit/api/quotes')
+      .then((res) => res.json())
+      .then((data) => {
+        const randomNumber = Math.floor(Math.random() * data.length);
+        setQuotes(data[randomNumber]);
+      });
+  };
+
+  useEffect(() => {
+    getQuotes();
+  }, []);
+
+  useEffect(() => {
+    textRef.current.style.color = colors[Math.floor(Math.random() * 8)];
+  }, [quotes]);
+
+  return (
+    <div className="quote">
+      <h2>Get motivated with new quote</h2>
+      <p className="quote-body" ref={textRef}>
+        {quotes.text}
+      </p>
+      <strong>
+        {' '}
+        ~
+        {quotes.author || 'anonyme'}
+      </strong>
+      <div className="btnContainer">
+        <button
+          type="button"
+          className="btn btn-warning"
+          onClick={getQuotes}
+        >
+          Get Quote
+        </button>
+        <a
+          href={`http://twitter.com/intent/tweet?text=${quotes.text}`}
```

Comment:
Please ensure that the general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).

Please `align center` as in the original design. 

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/165527078-d2a72349-326b-404e-80e2-b03674cd15de.png) | ![image](https://user-images.githubusercontent.com/10332499/165526956-fe0f6a58-0e00-4265-8392-4188a6e3b7e3.png)


Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/165527467-e83854ab-d2e2-439e-a03d-a944fc22ef7d.png) | ![image](https://user-images.githubusercontent.com/10332499/165527340-c9b76bad-f345-4078-b394-1fefd7f86922.png)

Please also ensure to include `footer` in all the pages according to the project requirement.

Reply:
Hi @micheaol ! Thank you for your review, but I wanted to ask you about the footer. I don`t read anywhere that we have to create a footer for this project. Can you confirm this is optional? Thank you in advance!

Reply:
@myelin0,

The project requirement states that your implementation should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). which includes everything in the wireframe.

I hope this helps.